types/underscore/index.d.ts

```diff
@@ -3532,9 +3518,9 @@ declare module _ {
          */
         findKey<V extends object>(
             object: V,
-            iteratee?: ObjectIteratee<V, boolean>,
+            iteratee?: Iteratee<V, boolean, TypeOfCollection<V, any>>,
             context?: any
-        ): string | undefined;
+        ): Strings<keyof V> | undefined;
```

Comment:
This is smart, but don't you mean `Extract<keyof V, string> | undefined`?

Reply:
I do mean that, thanks!